pkg/flink/config/sql_keywords.go

```diff
@@ -4,565 +4,508 @@ import (
 	"github.com/confluentinc/cli/v3/pkg/types"
 )
 
-// These are all Flink SQL reserved keywords extracted from https://nightlies.apache.org/flink/flink-docs-release-1.16/docs/dev/table/sql/overview/
-
 var SQLKeywords = types.NewSet(
```

Comment:
This will be replaced by the language server in the future, right?

Reply:
Yes, we're already looking for ways to do syntax highlighting with the language service!